libs/ledgerjs/packages/hw-app-exchange/src/Exchange.ts

```diff
@@ -98,6 +112,18 @@ export function createExchange(
   return new Exchange(transport, resolveTransactionType(transactionType, version), transactionRate);
 }
 
+export async function decodePayloadProtobuf(buffer: Buffer): Promise<PayloadCore> {
```

Comment:
This function should be in its own file.
It would add more readability in each files concerns.